### ISSUE TITLE
Implement user's custom text filters

### ISSUE THREAD

**Issue by cdmnky73** [mastodon/mastodon-ios#223]
Currently, the app doesn't hide any posts that may otherwise be filtered on desktop through a user's custom post text filters

They could either be removed completely, or replaced with a with text such as "post removed by filter"

**Comment by Danie10** [comment]
If it's the same thing I'm thinking of (the custom column on desktop with some set hashtags to filter on) then I'd also like to see this view available, even if we must set it on the app (ie. it does not pull through from desktop). It's something I use a lot to watch for mentions of certain topics from profiles I don't yet follow.

**Comment by MainasuK** [comment]
The "Filters" is implemented in the app. Please check the filter setting and make sure the timeline type you want to filter out is selected. Also, please unchecked the "Whole word" option if the post language can not be tokenized.

If issue still not resolved. Please provide some example filters then I can take a look.

**Comment by Danie10** [comment]
OK I see a Home feed with whoever I am following, and then a Search feed view, but you have to keep typing the search hashtags etc into search for it to filter. I was thinking more of the one view we have on a desktop column where we can set a filter condition like "any of these" and have #valheim #hubzilla #irc, and that stays as a saved filter view to quickly look at.

The existing search view seems to be picking up only the last hashtag. Ideally would be great to swipe between Home feed, Filtered View, and Notifications views, with a saved filtered view. It's the only way I'm alerted quickly to things I'm interested in, for people I'm not following yet. Maybe a saved filter in the filter view, that allows for multiple "any of" tags?

**Comment by MainasuK** [comment]
> OK I see a Home feed with whoever I am following, and then a Search feed view, but you have to keep typing the search hashtags etc into search for it to filter. I was thinking more of the one view we have on a desktop column where we can set a filter condition like "any of these" and have #valheim #hubzilla #irc, and that stays as a saved filter view to quickly look at.
> 
> The existing search view seems to be picking up only the last hashtag. Ideally would be great to swipe between Home feed, Filtered View, and Notifications views, with a saved filtered view. It's the only way I'm alerted quickly to things I'm interested in, for people I'm not following yet. Maybe a saved filter in the filter view, that allows for multiple "any of" tags?

Maybe that "Filter" is not what you want. The filter is to hidden the post content match the rules. What you want is maybe the "List". But the v1.0.0 is not with that feature yet.

**Comment by Danie10** [comment]
Yes thanks, not a filter view, but a list view. OK would be happy if it will be  pending feature for a later release.
